lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -50,8 +50,7 @@ describe('Fraggle Rock API', () => {
       expect(accessibility.score).toBeLessThan(1);
 
       const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
-      // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`73`);
+      expect(auditResults.map(audit => audit.id)).toMatchSnapshot();
```

Comment:
can some of these be removed? Like is this assertion accomplishing anything anymore?

Reply:
I personally think it's a good guard against regressions that remove snapshot support for certain audits.